snap/snapcraft.yaml

```diff
@@ -8,20 +8,43 @@ summary: The easiest, most secure way to use WireGuard and 2FA
 description: |
   A secure network that just works Zero config VPN.
 
-  Installs on any device in minutes, manages firewall rules for you, and works
-  from anywhere.
+  Installs on any device in minutes, manages firewall rules for you, and works from anywhere.
 
-  ## Confinement
+  After installing this snap, visit the Tailscale documentation at https://tailscale.com/kb/1017/install to get started.
+
+  **Confinement**
```

Comment:
Note: we can't use headings, `inline code`, or `[](urls)` in snap store markdown, hence some slightly odd markup here.

Reply:
Actually I think we can use inline code.